app/api/attendees.py

```diff
@@ -52,14 +53,26 @@ class Meta:
                          schema='EventSchema',
                          type_='event')
 
-class AttendeeList(ResourceList):
+
+class AttendeeListPost(ResourceList):
     """
-    List and create Attendees
+       List and create Attendees through direct URL
     """
 
+    @jwt_required
     def before_post(self, args, kwargs, data):
-        require_relationship(['event'], data)
+        require_relationship(['ticket', 'event'], data)
```

Comment:
@poush @enigmaeth ... This is a bit problematic here ... You're right @poush .. Also, make it such that attendees list shows only attendees linked to a ticket.